src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--month-length--30 calendar--start-day--fri">
+      <div class="calendar__month">
+        <div class="calendar__days"></div>
```

Comment:
no need for calendar__month wrapper, just calendar block full of calendar__day elements inside

```suggestion
    <div class="calendar calendar--month-length--30 calendar--start-day--fri">
        <div class="calendar__day"></div>
```

Reply:
But then everything breaks domn. I need to transfer the properties of .calendar to .body?